### PR TITLE
Fix Issue #412

### PR DESCRIPTION
**Pull request by AndreaGon**
The Clock Activity can now be in fullscreen mode. Any issues, please let me know!

Fix412

### CODE REVIEW

File: activities/Clock.activity/js/activity.js [hunk 1]
```diff
@@ -94,8 +94,27 @@ define(["sugar-web/activity/activity","sugar-web/env","sugar-web/graphics/radiob
                 that.updateSizes();
                 that.drawBackground();
             };
+            // Switch to full screen when the full screen button is pressed
+            document.getElementById("fullscreen-button").addEventListener('click', function() {
+              document.getElementById("main-toolbar").style.display = "none";
+              document.getElementById("canvas").style.top = "0px";
+              document.getElementById("unfullscreen-button").style.visibility = "visible";
```
Comment (FreddieN) [inline activities/Clock.activity/js/activity.js]: Please respect indentation of original file you use 2 spaces files uses 4. Same for below.

File: activities/Clock.activity/js/activity.js [hunk 2]
```diff
@@ -94,8 +94,27 @@ define(["sugar-web/activity/activity","sugar-web/env","sugar-web/graphics/radiob
...
+              that.drawBackground();
+            });
+
+            //Return to normal size
+            document.getElementById("unfullscreen-button").addEventListener('click', function() {
+              document.getElementById("main-toolbar").style.display = "block";
+              document.getElementById("canvas").style.top = "8.7%";
+              document.getElementById("unfullscreen-button").style.visibility = "hidden";
+              that.updateSizes();
+              that.drawBackground();
```
Comment (FreddieN) [inline activities/Clock.activity/js/activity.js]: Indentation not in sync with rest of file.

**Comment by llaske** [comment]
There is an issue when doing fullscreen/unfullscreen. See below:

Peek 05-12-2019 22-00

**Comment by AndreaGon** [comment]
I've fixed the issues in this new commit. Please let me know what you think!

**Comment by FreddieN** [comment]
https://i.imgur.com/07KU564.mp4

Issue appears to be fixed - sorry I did not test with a smaller screen size!

**Comment by llaske** [comment]
Good job. Thanks.

**Pull request merged**